test/test_other.py

```diff
@@ -2588,7 +2588,7 @@ def test_contrib_ports(self):
   def test_remote_ports(self):
     self.emcc(test_file('hello_world.c'), ['--use-port=emdawnwebgpu'])
 
-  @crossplatform
+  @crossplatform  # Note this only tests on platforms where FROZEN_CACHE isn't set.
```

Comment:
I don't think this comment is needed since you can see `test doesn't work with frozen cache` just below.